modules/javafx.controls/src/main/java/javafx/scene/control/DatePicker.java

```diff
@@ -147,11 +148,23 @@ public DatePicker(LocalDate localDate) {
 
         focusedProperty().addListener(o -> {
             if (!isFocused()) {
-                commitValue();
+                commitValueOnFocusLost();
             }
         });
     }
 
+    /**
+     * This is to address https://bugs.openjdk.org/browse/JDK-8303478
```

Comment:
I don't think this javadoc is needed, since it just links the issue one can also lookup when checking the git history

Reply:
Exactly. Remove this line and either add a comment describing what the method does, or since the name is self-explanatory, remove the comment block entirely.